css-layout-classes/styles.css

```diff
@@ -0,0 +1,29 @@
+.container {
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+  justify-content: center;
+}
+
+img,
+h2,
+p {
```

Comment:
`h2` and `p` are block-level elements, you shouldn't need to set `width: 100%` on them